lib/ex_unit/lib/ex_unit/callbacks.ex

```diff
@@ -2,8 +2,9 @@ defmodule ExUnit.Callbacks do
   @moduledoc ~S"""
   Defines ExUnit callbacks.
 
-  This module defines both `setup` and `setup_all` callbacks, as well as
-  the `on_exit/2`, `start_supervised/2` and `stop_supervised/1` functions.
+  This module defines the `setup/1`, `setup/2`, `setup_all/1` and
+  `setup_all/2` callbacks, as well as the `on_exit/2`, `start_supervised/2`
```

Comment:
Minor nitpick: oxford comma:

> This module defines the `setup/1`, `setup/2`, `setup_all/1`, and `setup_all/2` callbacks, as well as the `on_exit/2`, `start_supervised/2`, and `stop_supervised/1` functions.

Btw, would not be nice if ExDoc would know how to link `setup/1,2`? or `setup/1/2`?

Reply:
I think it will be a really good addition.